samples/mobile_run_test_app/Androidruntests/app/src/main/java/io/multy/android_run_tests/MainActivity.java

```diff
@@ -0,0 +1,33 @@
+package io.multy.android_run_tests;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+
+    // Used to load the 'native-lib' library on application startup.
+    static {
+        System.loadLibrary("multy_core_jnid");
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        TextView lib_version = (TextView) findViewById(R.id.lib_version);
+        lib_version.setText(version());
+        // Example of a call to a native method
+        TextView tv = (TextView) findViewById(R.id.sample_text);
+        tv.setText("In progress");
+        if (runTest() == 0) {
+            tv.setText("All TESTS OK");
+        } else {
+            tv.setText("Some TESTS FAILED. Look log");
```

Comment:
Please update message to "Some tests FAILED. Check logs."
Also it would be really nice to see list of tests that failed, not sure if that should be in this version, though.